src/librustdoc/core.rs

```diff
@@ -464,7 +464,7 @@ crate fn run_global_ctxt(
                 _ => continue,
             };
             for name in value.as_str().split_whitespace() {
-                let span = attr.name_value_literal_span().unwrap_or(attr.span());
+                let span = attr.name_value_literal_span().unwrap_or_else(|| attr.span());
```

Comment:
span() is really cheap, this just seems like churn.

Reply:
It has a small `match` (see [here](https://doc.rust-lang.org/nightly/nightly-rustc/src/rustc_ast/attr/mod.rs.html#548-553)), so I guess it counts as micro optimization?